UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -120,6 +120,8 @@ public partial class BodyPart
 			0
 		};
 
+		public DamageWeaknesses damageWeaknesses;
```

Comment:
gets modified by TOX  resistance mutation

Reply:
you aren't changing the instance tho. It should be public get, private set. Tox resistance mutation will modify a value inside ``DamageWeakness`` and I'd argue even that is bad because it breaks a pillar of OOP, encapsulation.